src/Visualization/Visualization.i

```diff
@@ -97,5 +98,14 @@ class Display3d {
 	Handle_AIS_InteractiveContext GetContext();
 	%feature("autodoc", "1");
 	void Test();
+
+	//%feature("autodoc", "1");
+    void ChangeRenderingParams(int RenderingMethod,
+                               int RaytracingDepth,
+                               bool IsShadowEnabled,

```

Comment:
Without having seen the generated code, I guess you missed to include Visualization.h or to declare the class Display3d